components/sensor/adc.rst

```diff
@@ -77,6 +77,10 @@ where you want to shut down the chip if the voltage is low when using a battery.
 
 To measure the VCC voltage, set ``pin:`` to ``VCC`` and make sure nothing is connected to the ``A0`` pin.
 
+.. note::
+
+    To avoid confusion: It measures the voltage at the chip, and not at the VCC pin of the board. It is usually static at 2.91V.
```

Comment:
Maybe we can elaborate on `at the chip` more. This option measures the logic supply voltage to the chip (around 3.3V).

What I don't quite understand is why it would usually be "static at 2.91V". I don't see that here, and I suspect that depends on the power supply.

Reply:
i got the 2.91V on two different device models (`nodemcuv2`, `d1_mini`) with different power supplies (`USB`, `6V VCC`).
i can do some more testing, but i'm also interested what your devices show.

Reply:
looks like a case of extrapolating from incomplete data. my ESP8285 (tuya power socket) reports slighly varying values around 3.46V.